node_modules/oae-tenants/lib/api.js

```diff
@@ -456,6 +457,7 @@ var _updateCachedTenant = function(tenantAlias, callback) {
  * @param  {String}     host                    The host on which this tenant will be proxying (e.g. oae.cam.ac.uk or oae.gatech.edu)
  * @param  {Object}     [opts]                  Optional arguments
  * @param  {String}     [opts.emailDomain]      The email domain for this tenant
+ * @param  {String}     [opts.countryCode]      The ISO-3166-1 country code of the country that represents the tenant
```

Comment:
We document `ISO-3166-1` but are requiring `iso-3166-2`. Is there any significant difference ?